.github/workflows/release_staging.yml

```diff
@@ -9,11 +9,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
+        uses: actions/checkout@v4
```

Comment:
we should be using the sha as it is the security best practice. So, can we revert these changes? :)